### PR TITLE
Store normandy id fixes #910

### LINKED ISSUE #910: Add Field for the Normandy Recipe ID Number
Adding a field to enter the Normandy Recipe number makes connecting from the Experiment to the Normandy JSON much simpler.

Having this number makes it simpler for admins to move between Experimenter and Normandy.

This would be editing after it is in "Ready to Ship" state - so may be blocked on #874

### PR DESCRIPTION
**Pull request by jaredkerim**

### CODE REVIEW

File: app/experimenter/experiments/migrations/0034_auto_20190319_2119.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            model_name="experiment",
+            name="normandy_id",
+            field=models.PositiveIntegerField(blank=True, null=True),
+        ),
+        migrations.AlterField(
+            model_name="experiment",
+            name="client_matching",
+            field=models.TextField(
+                blank=True,
+                default="Prefs:\n\nStudies:\n\nAny additional filters:\n    ",
```
Comment (mythmon) [inline app/experimenter/experiments/migrations/0034_auto_20190319_2119.py]: This doesn't look right. Did this get left out of a previous PR?

Comment (jaredkerim) [reply app/experimenter/experiments/migrations/0034_auto_20190319_2119.py]: It suuuuurrreeeee did.  https://github.com/mozilla/experimenter/pull/1068

Comment (jaredkerim) [reply app/experimenter/experiments/migrations/0034_auto_20190319_2119.py]: Maybe we should add a step to continuous integration to check that no migrations got missed.

Comment (jaredkerim) [reply app/experimenter/experiments/migrations/0034_auto_20190319_2119.py]: Filed #1079

Comment (mythmon) [reply app/experimenter/experiments/migrations/0034_auto_20190319_2119.py]: I think we should, yeah. We [did it on Normandy](https://github.com/mozilla/normandy/pull/1591) recently, and it was a pretty easy check.

Do you want to leave this here and merge as is, or do you want to do something to fix this?

Comment (jaredkerim) [reply app/experimenter/experiments/migrations/0034_auto_20190319_2119.py]: Well It has to go in and I don't see the point of splitting it into its own tiny PR so I'm fine with this as is.

**Pull request merged**